docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Simulation output shape (y)</th>\n",
+       "      <td>(500, 5)</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># hold-out set samples (test_set_size)</th>\n",
+       "      <td>100</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Do hyperparameter search (param_search)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Type of hyperparameter search (search_type)</th>\n",
+       "      <td>random</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># sampled parameter settings (param_search_iters)</th>\n",
+       "      <td>20</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scale data before fitting (scale)</th>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Scaler (scaler)</th>\n",
+       "      <td>StandardScaler</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction before fitting (reduce_dim)</th>\n",
+       "      <td>False</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Dimensionality reduction method (dim_reducer)</th>\n",
+       "      <td>PCA</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Cross-validation strategy (cross_validator)</th>\n",
+       "      <td>KFold</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th># parallel jobs (n_jobs)</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ad3b220743a840d7943170a5f300ea71",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Initializing:   0%|          | 0/11 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Time taken: 27.888551950454712 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sklearn.model_selection import KFold\n",
+    "\n",
+    "start = time.time()\n",
+    "\n",
+    "em = AutoEmulate()\n",
+    "em.setup(X, y, cross_validator=KFold(n_splits=3))\n",
+    "em.compare()\n",
+    "\n",
+    "end = time.time()\n",
+    "print(f\"Time taken: {end - start} seconds\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4) modify hyperparameter search\n",
+    "\n",
+    "As mentioned above, we are fitting a lot of models when searching for the best hyperparameters, it's therefore recommended to focus on a few models of interest to maximise performance."
```

Comment:
I would move the explanations on hyperparameters down here

Reply:
also, for this section the reader doesn't really have a comparison for what time hyperparameter search would've required if run on all models. I assume it is not feasible to actually measure but can we provide a ballpark number here to really illustrate the gain?

Reply:
did that, and added a bit of text and how to get a ballpark figure for hyperparameter search computation time.